keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -258,6 +258,10 @@ At a high-level, DRA with structured parameters takes the following form:
   schedule a pod on (as well as allocate resources from its `ResourceSlice`
   in the process).
 
+* Alternatively, a `ResourceClaim` may also directly reference a
```

Comment:
So, if users want to embed ResourceClaimParameters into ResourceClaim or generate ResourceClaimParameters manually, they may need to write some CEL expressions to filter out resources, right?

Reply:
Yes.

There's an ongoing debate between @johnbelamaric and @klueska whether asking users to write CEL expressions is going to be a good API. I suspect it is both: it's okay in some cases and too complex in others, so we'll need support for CRDs as an optional feature.